CODE_STYLE.md

```diff
@@ -2,3 +2,14 @@
 
 * [Ruby](https://github.com/bbatsov/ruby-style-guide)
 * [Objective-C](https://github.com/hyperoslo/objective-c-style-guide)
+* [JavaScript](https://github.com/hyperoslo/eslint-config)
+
+## Stylus
+
+We have not yet chosen a style guide for Stylus, other than to decide that:
+
+* We *don't* use `;`.
+* We *do* use `{` and `}`.
```

Comment:
Me too, I would prefer to not use `{}` like we don't use `;`. There's just no need in my mind.